sdk/src/claim.rs

```diff
@@ -1018,10 +1019,11 @@ impl Claim {
     /// Verify claim signature, assertion store and asset hashes
     /// claim - claim to be verified
     /// asset_bytes - reference to bytes of the asset
-    pub async fn verify_claim_async<'a>(
+    pub(crate) async fn verify_claim_async<'a>(
```

Comment:
Since the whole Claim structure isn't public, we don't need to declare this crate private, but it doesn't hurt.

Reply:
I have started to make this explicit.  Sometimes things get exposed that we did not intend.

Reply:
sounds good